packages/yoroi-extension/app/components/common/Autocomplete.js

```diff
@@ -177,10 +175,7 @@ function Autocomplete({
             })}
           />
         </Box>
-        <CheckWrapper>
-          {done === true ? <SuccessIcon /> : null}
-          {Boolean(error) === true ? <ErrorIcon /> : null}
```

Comment:
There's no error icon anymore?

Reply:
It is removed from all inputs in design.